core/src/test/resources/csv-inputs/array.csv

```diff
@@ -0,0 +1,2 @@
+:ID|name:STRING[]
```

Comment:
there are many CSVs now, is that intentional?

Reply:
Something happened when I switched from branch `4.1` where I was working on before your last comments, I removed the files. Thank you!